ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -426,12 +426,16 @@ public boolean maybeFailedDueToOfflineReplicas() {
     }
     if (hasFailedOnCrossColoNotFound() && allReplicaCount - totalNotFoundCount >= replicaSuccessTarget
         && totalOfflineReplicaCount > 0) {
+      Map<ReplicaState, List<ReplicaId>> replicasByState = getReplicasByState(null,
+          EnumSet.of(ReplicaState.BOOTSTRAP, ReplicaState.STANDBY, ReplicaState.LEADER, ReplicaState.INACTIVE,
+              ReplicaState.OFFLINE, ReplicaState.ERROR, ReplicaState.DROPPED));
       logger.info(
           "Terminating {} on {} due to disk down count and total Not_Found count from eligible replicas and some "
               + "other replicas being unavailable. CrossColoEnabled: {}, DiskDownCount: {}, TotalNotFoundCount: {}, "
-              + "TotalReplicaCount: {}, replicaSuccessTarget: {}, OfflineReplicaCount: {}, allReplicaCount: {} {}",
+              + "TotalReplicaCount: {}, replicaSuccessTarget: {}, OfflineReplicaCount: {}, allReplicaCount: {} {} "
+              + "replicasByState = {}",
           routerOperation, partitionId, crossColoEnabled, diskDownCount, totalNotFoundCount, totalReplicaCount,
-          replicaSuccessTarget, totalOfflineReplicaCount, allReplicaCount, getBlobIdLog());
+          replicaSuccessTarget, totalOfflineReplicaCount, allReplicaCount, getBlobIdLog(), replicasByState);
```

Comment:
there is already a class member called allDcReplicasByState, we should use that member.

Reply:
I missed that. Thanks.